cost-analyzer/templates/cost-analyzer-deployment-template.yaml

```diff
@@ -303,6 +303,10 @@ spec:
         - image: gcr.io/kubecost1/cost-model:prod-{{ $.Chart.AppVersion }}
         {{ end }}
           name: cost-model
+          ports:
+          - containerPort: 50052
+            name: grpcserver
```

Comment:
Is this one port for all services? I.e. if we had an assets service would it also be on this port?

Reply:
It will be upto you as to how you would want it because u can open another tcp port exclusive of this to listen to new request or have a new route with same grpc server. 

If we choose the latter, you would register it with the same tcp port
Like in here you would need another similar registering of service https://github.com/kubecost/kubecost-cost-model/pull/955/files#diff-c9127ce2d87ae5fe988d6b5f16b63f7ba022998d617dfbafd0381bfda6ceba41R2449|

Something along lines of https://stackoverflow.com/questions/52634217/access-multiple-grpc-services-over-the-same-connection

Once that's done you would need to have another route defined pointing to package of the new service so it would route the request appropriately .